Sources/FoundationEssentials/Decimal/Decimal+Conformances.swift

```diff
@@ -14,6 +14,130 @@
 internal import _ForSwiftFoundation
 #endif // FOUNDATION_FRAMEWORK
 
+/*
```

Comment:
Are we leaving these as comments intentionally?

Reply:
Nope, thanks. Will fix.